### ISSUE TITLE
415608AB - Skylanders: Spyro's Adventure

### ISSUE THREAD

**Issue by NeighborhoodNews** [xenia-project/game-compatibility#1518]
[Marketplace](https://marketplace.xbox.com/en-us/Product/Skylanders-Spyros-Adventure/66acd000-77fe-1000-9115-d802415608ab)

Tested on https://github.com/xenia-project/xenia/commit/637a8464a79d62d5dd7d381bfcf51aeccfb2988b

# Issues:
Xenia's icon in the taskbar changes to look like Spyro, but the emulator shows a black screen.

# Log:
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/4456555/xenia.zip)
# Screenshot(s):
![xeniascreenshot](https://user-images.githubusercontent.com/30245629/78903190-c8018c00-7a48-11ea-9e5f-3709e5e6b3db.PNG)

# Labels:
marketplace-nodownload
state-hang

**Comment by Oririr** [comment]
where download link

**Comment by Etokapa** [comment]
Tested on https://github.com/xenia-project/xenia/commit/9a74df491ffd0189ddd26bc2030e0eae6a775f4e

Started a new game. The game wants a Portal of Power after the initial cut scene.

:warning:You must set `scribble_heap=true`!

![sky (2)](https://user-images.githubusercontent.com/31154606/105533916-61392080-5cb2-11eb-90b0-befed4f26c0c.png)

<details>
  <summary>More Screenshots</summary>

![sky (1)](https://user-images.githubusercontent.com/31154606/105533927-65653e00-5cb2-11eb-994b-08e198448722.png)
![sky (3)](https://user-images.githubusercontent.com/31154606/105533928-65653e00-5cb2-11eb-992f-0fc694415953.png)

</details>

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/5857994/xenia.zip)

state-menus

**Comment by MewQ12** [comment]
Can you actually use a portal to play or is there no way to use a portal through Xenia?

**Comment by The1stGamer** [comment]
yeah the game needs a portal of power, any fix for this? i have the portal but cant use it :/

**Comment by KaosAron** [comment]
Same Problem

**Comment by Pizzabot3001** [comment]
do

> Tested on [xenia-project/xenia@9a74df4](https://github.com/xenia-project/xenia/commit/9a74df491ffd0189ddd26bc2030e0eae6a775f4e)
> 
> Started a new game. The game wants a Portal of Power after the initial cut scene.
> 
> ⚠️You must set `scribble_heap=true`!
> 
> ![sky (2)](https://user-images.githubusercontent.com/31154606/105533916-61392080-5cb2-11eb-90b0-befed4f26c0c.png)
> More Screenshots
> 
> ![sky (1)](https://user-images.githubusercontent.com/31154606/105533927-65653e00-5cb2-11eb-994b-08e198448722.png) ![sky (3)](https://user-images.githubusercontent.com/31154606/105533928-65653e00-5cb2-11eb-992f-0fc694415953.png)
> 
> [xenia.zip](https://github.com/xenia-project/game-compatibility/files/5857994/xenia.zip)
> 
> state-menus

can you say me how you have done it

**Comment by zenith2004** [comment]
> Tested on [xenia-project/xenia@9a74df4](https://github.com/xenia-project/xenia/commit/9a74df491ffd0189ddd26bc2030e0eae6a775f4e)
> 
> Started a new game. The game wants a Portal of Power after the initial cut scene.
> 
> ⚠️You must set `scribble_heap=true`!
> 
> ![sky (2)](https://user-images.githubusercontent.com/31154606/105533916-61392080-5cb2-11eb-90b0-befed4f26c0c.png)
> 
> More Screenshots
> [xenia.zip](https://github.com/xenia-project/game-compatibility/files/5857994/xenia.zip)
> 
> state-menus

does anyone have a way to fix this? I'm getting the same error.

**Comment by Kapitan2015** [comment]
> Tested on [xenia-project/xenia@9a74df4](https://github.com/xenia-project/xenia/commit/9a74df491ffd0189ddd26bc2030e0eae6a775f4e)
> 
> Started a new game. The game wants a Portal of Power after the initial cut scene.
> 
> ⚠️You must set `scribble_heap=true`!
> 
> ![sky (2)](https://user-images.githubusercontent.com/31154606/105533916-61392080-5cb2-11eb-90b0-befed4f26c0c.png)
> 
> More Screenshots
> [xenia.zip](https://github.com/xenia-project/game-compatibility/files/5857994/xenia.zip)
> 
> state-menus

Hi, what portal did you use for it to work ? Xbox? Wii?

**Comment by Reddin012** [comment]
> Tested on [xenia-project/xenia@9a74df4](https://github.com/xenia-project/xenia/commit/9a74df491ffd0189ddd26bc2030e0eae6a775f4e)
> 
> Started a new game. The game wants a Portal of Power after the initial cut scene.
> 
> ⚠️You must set `scribble_heap=true`!
> 
> ![sky (2)](https://user-images.githubusercontent.com/31154606/105533916-61392080-5cb2-11eb-90b0-befed4f26c0c.png)
> 
> More Screenshots
> [xenia.zip](https://github.com/xenia-project/game-compatibility/files/5857994/xenia.zip)
> 
> state-menus

Ok but where can we set it to true where can I find scribble_heap=true

**Comment by BigAssSalmon** [comment]
is there a way to connect the original portal of power is this all we got
